src/Sfx/App/Scripts/Controllers/AppViewController.ts

```diff
@@ -70,25 +73,36 @@ module Sfx {
             this.$scope.upgradeProgressUnhealthyEvaluationsListSettings = this.settings.getNewOrExistingUnhealthyEvaluationsListSettings("upgradeProgressUnhealthyEvaluations");
             this.$scope.appEvents = this.data.createApplicationEventList(this.appId);
 
+            this.$scope.networkListSettings = this.settings.getNewOrExistingListSettings("networks", ["networkDetail.name"], [
+                new ListColumnSettingForLink("networkDetail.name", "Network Name", item => item.viewPath),
+                new ListColumnSetting("networkDetail.type", "Network Type"),
+                new ListColumnSetting("networkDetail.addressPrefix", "Network Address Prefix"),
+                new ListColumnSetting("networkDetail.status", "Network Status"),
+            ]);
+            this.$scope.clusterManifest = new ClusterManifest(this.data);
+            this.$scope.networks = new  NetworkOnAppCollection(this.data, this.appId);
             this.refresh();
         }
 
         protected refreshCommon(messageHandler?: IResponseMessageHandler): angular.IPromise<any> {
-            return this.data.getApp(this.appId, true, messageHandler).then(data => {
-                this.$scope.app = data;
-
-                return this.$scope.app.health.refresh(messageHandler).then(() => {
-                    this.$scope.deployedApplicationsHealthStates = this.$scope.app.health.deployedApplicationHealthStates;
-
-                    if (this.$scope.app.health.deploymentsHealthState.text !== HealthStateConstants.OK) {
-                        this.tabs["deployments"].superscriptInHtml = () => {
-                            return `<image class="tab-superscript-badge" src="images/${this.$scope.app.health.deploymentsHealthState.badgeClass}.svg"></image>`;
-                        };
-                    } else {
-                        this.tabs["deployments"].superscriptInHtml = null;
-                    }
-                });
-            });
+            return this.$q.all([
+                this.data.getApp(this.appId, true, messageHandler).then(data => {
+                    this.$scope.app = data;
+
+                    return this.$scope.app.health.refresh(messageHandler).then(() => {
+                        this.$scope.deployedApplicationsHealthStates = this.$scope.app.health.deployedApplicationHealthStates;
+
+                        if (this.$scope.app.health.deploymentsHealthState.text !== HealthStateConstants.OK) {
+                            this.tabs["deployments"].superscriptInHtml = () => {
+                                return `<image class="tab-superscript-badge" src="images/${this.$scope.app.health.deploymentsHealthState.badgeClass}.svg"></image>`;

```

Comment:
Is there a better way than returning a string? Can you use an object representing the image tag instead?